Client/Frontend/Browser/BrowserLocationView.swift

```diff
@@ -171,10 +171,10 @@ class BrowserLocationView: UIView {
     }
 
     private func updateTextWithURL() {
-        if let httplessURL = url?.absoluteStringWithoutHTTPScheme(), let baseDomain = url?.baseDomain() {
+        if let protocollessURL = url?.absoluteStringWithoutProtocol(), let baseDomain = url?.baseDomain() {
```

Comment:
This isn't the right change to be making; this code affects the URL bar.

Take a look at `TopSitesPanel.swift`, `TopSitesDataSource.createTileForSite`. That's where you should be using something other than the raw URL.